sjb/config/test_cases/test_branch_openshift_ansible_logging.yml

```diff
@@ -52,9 +52,28 @@ extensions:
       title: "determine the release commit for origin images and version for rpms"
       repository: "origin"
       script: |-
+        # is logging using master or a release branch?
+        pushd "/data/src/github.com/openshift/origin-aggregated-logging/"
+        curbranch=$( git branch | awk '/^[*]/ {print $2}' )
+        popd
         jobs_repo="/data/src/github.com/openshift/aos-cd-jobs/"
-        git log -1 --pretty=%h >> "${jobs_repo}/ORIGIN_COMMIT"
-        ( source hack/lib/init.sh; os::build::rpm::get_nvra_vars; echo "-${OS_RPM_VERSION}-${OS_RPM_RELEASE}" ) >> "${jobs_repo}/ORIGIN_PKG_VERSION"
+        if [ "${curbranch}" = master ] ; then
+           git log -1 --pretty=%h >> "${jobs_repo}/ORIGIN_COMMIT"
+           ( source hack/lib/init.sh; os::build::rpm::get_nvra_vars; echo "-${OS_RPM_VERSION}-${OS_RPM_RELEASE}" ) >> "${jobs_repo}/ORIGIN_PKG_VERSION"
+        else
```

Comment:
If you're going to require that the branch start with `release-` I would make this:

```sh
elif [[ "${curbranch}" =~ ^release-* ]]; then
```